### PR TITLE
[Card#39]-Added delete leave feature remote_api & local_db

### PR DESCRIPTION
**Pull request by HarshwardhanSingh**
Allows users to delete leave from local database as well as from remote api.

Trello card: https://trello.com/c/gX8eik4C

### CODE REVIEW

**Comment by deepak** [comment]
same comments as https://github.com/code-mancers/leave-tracker/pull/36

File: app/controllers/leaves_controller.rb [hunk 1]
```diff
@@ -52,7 +52,14 @@ def update
   end
 
   def destroy
-    response = @client.remove_leave(params[:calendar_id], params[:id])
+    leave = Leave.find(params[:id])
+    calendarleaves = Calendarleave.where(leave_id: leave.id)
+    calendarleaves.each do |calendarleave|
+      calendar = Calendar.find(calendarleave.calendar_id)
+      @client.remove_leave(calendar.google_id, calendarleave.leave_google_id)
```
Comment (deepak) [inline app/controllers/leaves_controller.rb]: what happens if the api call to google timeouts or fails ?

Comment (HarshwardhanSingh) [inline app/controllers/leaves_controller.rb]: will solve this in different PR

File: config/routes.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@
   get '/dashboard', to: 'pages#dashboard', as: 'dashboard'
 
   resources :calendars, except: [:destroy] do
-    resources :leaves, except: [:new, :create, :edit, :update]
+    resources :leaves, except: [:new, :create, :edit, :update, :destroy]
```
Comment (deepak) [inline config/routes.rb]: do no need this list of actions anymore. have added everything

**Pull request merged**